soundcard/mediafoundation.py

```diff
@@ -403,16 +412,17 @@ class _Microphone(_Device):
 
     """
 
-    def __init__(self, device):
+    def __init__(self, device, isloopback=False):
         self._id = device._id
+        self.isloopback = isloopback
 
     def __repr__(self):
         return f'<Microphone {self.name} ({self.channels} channels)>'
```

Comment:
We should include a marker that this is a loopback device here.

Reply:
Should I also add this property in the Linux code and also change the naming (in the __repr__ function) to Loopback instead of monitor to keep the naming consistent?

Reply:
Very good idea!

Reply:
Done. Also added the property to macOS code to keep the consistency.